tests/unittest/others/test_layer.py

```diff
@@ -23,11 +23,12 @@
 import torch
 import tensorrt as trt
 # isort: on
-from functional.torch_ref import (attention_qkvpacked_ref, group_rms_norm_ref,
-                                  mamba2_ref, mamba_ref, recurrent_ref)
 from parameterized import parameterized
 from polygraphy.backend.trt import (CreateConfig, EngineFromNetwork, Profile,
                                     TrtRunner)
+from test_trt.functional.torch_ref import (attention_qkvpacked_ref,
+                                           group_rms_norm_ref, mamba2_ref,
+                                           mamba_ref, recurrent_ref)
```

Comment:
Let's move torch_ref.py from functional directory to utils directory. Then, the tests folder hierarchy will be:


```
tests
    _torch
    other
    trt
    utils
        torch_ref.py
```